apps/src/JSInterpreter.js

```diff
@@ -1141,19 +1142,23 @@ JSInterpreter.prototype.handlePauseContinue = function () {
     this.paused = true;
     this.nextStep = StepType.RUN;
   }
+  this.studioApp.reduxStore_.dispatch(runState.setIsDebugging(this.paused));
 };
 
 JSInterpreter.prototype.handleStepOver = function () {
   this.paused = true;
   this.nextStep = StepType.OVER;
+  this.studioApp.reduxStore_.dispatch(runState.setIsDebugging(this.paused));
 };
 
 JSInterpreter.prototype.handleStepIn = function () {
   this.paused = true;
   this.nextStep = StepType.IN;
+  this.studioApp.reduxStore_.dispatch(runState.setIsDebugging(this.paused));
 };
 
 JSInterpreter.prototype.handleStepOut = function () {
   this.paused = true;
   this.nextStep = StepType.OUT;
+  this.studioApp.reduxStore_.dispatch(runState.setIsDebugging(this.paused));
```

Comment:
I think this whole thing needs a refactor at some point but this may not be the PR to do it, anyway.